src/app/Processor/ProcessorEdit/ProcessorEdit.css

```diff
@@ -1,35 +1,6 @@
-.processor-edit__page-section {
-  --processor-edit--spacing: var(--pf-global--spacer--lg);
-  position: relative;
-}
-
-.processor-edit__container {
-  position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-}
-
-.processor-edit__outer-wrap {
-  min-height: 0;
-  /*position: relative;*/
-}
-
-.processor-edit__inner-wrap {
-  /*position: static;*/
-  min-height: 0;
-}
-
-.processor-edit__content-wrap {
-  overflow-x: hidden;
-  overflow-y: auto;
-  word-break: break-word;
-}
-
```

Comment:
Old sticky actions layout used within `ProcessorEdit`, now made generic and moved to the new layout component.